client/post-editor/editor-seo/accordion.jsx

```diff
@@ -0,0 +1,84 @@
+/**
+ * External dependencies
+ */
+import React, { Component, PropTypes } from 'react';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import Accordion from 'components/accordion';
+import AccordionSection from 'components/accordion/section';
+import CountedTextarea from 'components/forms/counted-textarea';
+import Gridicon from 'components/gridicon';
+import InfoPopover from 'components/info-popover';
+import TokenField from 'components/token-field';
+import { getSiteSlug } from 'state/sites/selectors';
+
+class SeoAccordion extends Component {
+	constructor() {
+		super();
+
+		this.displayName = 'EditorSeoAccordion';
+	}
+
+	render() {
+		const { translate } = this.props;
+		// Temporary placeholder chips for design review
+		const sampleChips = [ 'Post Title', 'Site Title' ];
+
+		return (
+			<Accordion
+				title={ translate( 'Advanced SEO' ) }
+				icon={ <Gridicon icon="search" /> }
+				className="editor-drawer__editor-seo"
+			>
+				<AccordionSection>
+					<span className="editor-drawer__label-text">
+						{ translate( 'Meta Title' ) }
+						<InfoPopover position="top left">
+							{ translate(
+								'The format for the title as it will appear in search engines. ' +
+								'{{a}}Edit{{/a}}',
+								{
+									components: {
+										a: <a target="_blank" href={ this.props.siteSlug } />
+									}
+								}
+							) }
+						</InfoPopover>
+					</span>
+					<TokenField value={ sampleChips } disabled={ true } />
```

Comment:
FYI `={ true }` can be omitted from JSX attributes, e.g. `<TokenField disabled />`